sentinelhub/areas.py

```diff
@@ -475,3 +484,152 @@ def _make_split(self):
+                                                   utm_zone=str(utm_cell_prop['zone']).zfill(2),
+                                                   utm_row=utm_cell_prop['row'],
+                                                   direction=utm_cell_prop['direction'],
+                                                   index=index,
+                                                   index_x=i,
+                                                   index_y=j))
+                        index += 1
+
+    def get_bbox_list(self, buffer=None):
+        """ Get list of bounding boxes.
+
+        The CRS is fixed to the computed UTM CRS. This BBox splitter does not support reducing size of output
+        bounding boxes
+
+        :param buffer: A percentage of each BBox size increase. This will cause neighbouring bounding boxes to overlap.
+        :type buffer: float or None
+        :return: List of bounding boxes
+        :rtype: list(BBox)
+        """
+        return super().get_bbox_list(buffer=buffer)
+
+
+class UtmGridSplitter(BaseUtmSplitter):
+    """ Splitter that returns bounding boxes of fixed size aligned to the UTM MGRS grid """
+
+    def _get_utm_polys(self):
+        """ Find UTM grid zones overlapping with input area shape
+
+        :return: List of geometries and properties of UTM grid zones overlapping with input area shape
+        """
+        # file downloaded from faculty.baruch.cuny.edu/geoportal/data/esri/world/utmzone.zip
+        utm_grid_filename = os.path.join(os.path.dirname(__file__), '.utmzones.geojson')
+
+        if not os.path.isfile(utm_grid_filename):
+            raise IOError('UTM grid definition file does not exist: %s' % os.path.abspath(utm_grid_filename))
+
+        with open(utm_grid_filename) as utm_grid_file:
+            utm_grid = json.load(utm_grid_file)['features']
+
+        utm_geom_list = [shapely.geometry.shape(utm_zone['geometry']) for utm_zone in utm_grid]
+        utm_prop_list = [dict(zone=utm_zone['properties']['ZONE'],
+                              row=utm_zone['properties']['ROW_'],
+                              direction='N' if utm_zone['properties']['ROW_'] >= 'N' else 'S') for utm_zone in utm_grid]
+
+        return [(utm_geom, utm_prop) for utm_geom, utm_prop in zip(utm_geom_list, utm_prop_list)]
+
+
+class UtmZoneSplitter(BaseUtmSplitter):
+    """ Splitter that returns bounding boxes of fixed size aligned to the equator and the UTM zones.
+    """
+    LNG_MIN, LNG_MAX, LNG_UTM = -180, 180, 6
+    LAT_MIN, LAT_MAX, LAT_EQ = -80, 84, 0
+
+    def _get_utm_polys(self):
+        """ Find UTM zones overlapping with input area shape
+
+        The returned geometry corresponds to the a triangle ranging from the equator to the north/south pole
+
+        :return: List of geometries and properties of UTM zones overlapping with input area shape
+        """
+        utm_geom_list = [Polygon([(lng, lat[0]), (lng, lat[1]), (lng + self.LNG_UTM, lat[0]),
```

Comment:
I only noticed this now, but cooridnates are in wrong order. That is why you had to use envelope or convex_hull. The right order is
```
(lng, lat[0]), (lng, lat[1]), (lng + self.LNG_UTM, lat[1]), (lng + self.LNG_UTM, lat[0]), (lng, lat[0])
```

Reply:
good catch. changed